### PR TITLE
Added scan path argument

### PR DESCRIPTION
**Pull request by rs--**
`scan_path` argument allows the user to specify which directory to perform a CodeQL scan on.

### CODE REVIEW

**Comment by ghost** [comment]
Welcome @rs--! <br><br>It looks like this is your first PR to <a href='https://github.com/openenclave/openenclave-security'>openenclave/openenclave-security</a> 🎉🎉. Please refer to our [pull request process documentation](https://github.com/openenclave/test-infra/blob/master/docs/owners.md#the-code-review-process) to help your PR have a smooth ride to approval. <br><br>You will be prompted by a bot to use commands during the review process. Do not be afraid to follow the prompts! It is okay to experiment. [Here is the bot commands documentation](https://go.k8s.io/bot-commands). <br><br>You can also check if openenclave/openenclave-security has [its own contribution guidelines](https://github.com/openenclave/openenclave-security/tree/master/CONTRIBUTING.md). <br><br>Thank you, and welcome to openenclave. :smiley:

File: src/static/codeql/scripts/codeql.py
Comment (mrragava) [inline src/static/codeql/scripts/codeql.py]: Lets use logger.info instead of print.

Comment (mrragava) [reply src/static/codeql/scripts/codeql.py]: f"...{scan_path}..." would be good.

File: scripts/run.sh
Comment (mrragava) [inline scripts/run.sh:10]: This is nice, Makes it generic to run on any project like oeedger8r or other OE consumers.
We might have to add another suite (oe-codeql-security-queries-only.qls) to scan using codeql queries alone without Open Enclave SDK specific queries.

File: src/static/codeql/scripts/codeql.py
Comment (rs--) [reply src/static/codeql/scripts/codeql.py]: I kept the verbosity for easier debug IF the `scan_path` argument is used. Less verbose logging is added in lines `429` and `431`

**Review by mrragava — APPROVED** [review]

**Comment by ghost** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/openenclave/openenclave-security/pull/5#pullrequestreview-591823687" title="Approved">mrragava</a>*, *<a href="https://github.com/openenclave/openenclave-security/pull/5#" title="Author self-approved">rs--</a>*

The full list of commands accepted by this bot can be found [here](?repo=openenclave%2Fopenenclave-security).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/openenclave/openenclave-security/blob/main/OWNERS)~~ [mrragava]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by rs--** [comment]
@mrragava Can you comment `/lgtm` please? I think the bot needs an approval + the `lgtm` tag prior to merging.

**Pull request merged**
